src/lib.rs

```diff
@@ -322,6 +324,22 @@ impl Client {
         watch::Watch::new(self, query)
     }
 
+    /// Disables [`Row`] types validation against the database schema.
+    /// Validation is enabled by default.
+    ///
+    /// # Warning
+    ///
+    /// While disabled validation will result in increased performance,
+    /// this mode is intended to be used for testing purposes only,
+    /// and only in scenarios where schema mismatch issues are irrelevant.
+    ///
+    /// ***DO NOT*** disable validation in your production code or tests
+    /// unless you are 100% sure why you are doing it.
+    pub fn with_disabled_validation(mut self) -> Self {
```

Comment:
Let's use `with_validation(false)` to simplify cases when this parameter is passed from configuration (to avoid `client = client.with...` code)